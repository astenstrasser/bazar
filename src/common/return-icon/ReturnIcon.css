.return-icon {
    text-decoration: none;
    color: #003559;
    font-weight: 600;
    display: flex;
    align-items: center;
}

.return-icon img {
    width: 32px;
}
