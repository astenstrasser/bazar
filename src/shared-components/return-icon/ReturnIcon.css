.return-icon {
  text-decoration: none;
  color: #003559;
  font-weight: 600;
  display: flex;
  align-items: center;
  margin-top: 1rem;
}

.return-icon img {
  width: 32px;
}

.return-icon--container{
  display: inline-block;
}
