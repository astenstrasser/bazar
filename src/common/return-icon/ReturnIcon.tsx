import React from "react";
import ArrowIcon from "../../assets/left-arrow.svg";
import "./ReturnIcon.css";

function ReturnIcon(props: { reference: string }): React.ReactElement {
  return (
    <>
      <a href={props.reference} className="return-icon">
        <img src={ArrowIcon} alt="Seta para retornar à página anterior" />
        Voltar
      </a>
    </>
  );
}

export default ReturnIcon;
