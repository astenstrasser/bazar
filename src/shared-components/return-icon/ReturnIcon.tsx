import React from "react";
import ArrowIcon from "../../assets/left-arrow.svg";
import "./ReturnIcon.css";

function ReturnIcon(props: { reference: string }): React.ReactElement {
  return (
    <div className="return-icon--container">
      <a href={props.reference} className="return-icon">
        <img src={ArrowIcon} alt="Retornar à página anterior" />
        Voltar
      </a>
    </div>
  );
}

export default ReturnIcon;
