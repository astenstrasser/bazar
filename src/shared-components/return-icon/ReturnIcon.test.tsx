import React from "react";
import { render } from "@testing-library/react";
import ReturnIcon from "./ReturnIcon";

test("renders snapshot", () => {
  const { container } = render(<ReturnIcon reference={"/"} />);
  expect(container).toMatchSnapshot();
});
