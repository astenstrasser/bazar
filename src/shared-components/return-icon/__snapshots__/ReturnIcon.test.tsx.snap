// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<div>
  <div
    class="return-icon--container"
  >
    <a
      class="return-icon"
      href="/"
    >
      <img
        alt="Retornar à página anterior"
        src="left-arrow.svg"
      />
      Voltar
    </a>
  </div>
</div>
`;
